### DISCUSSION TITLE
No document for Header Filters

### DISCUSSION THREAD

**Discussion by ukizui** [Comcast/react-data-grid#2857] [Q&A]
Although there is an example for header filters, I can't find any document for it.

I'd like to use this feature, but I don't know how to use it as there is no document.

I could find document for version 6 below, but its code sandbox was also broken like a screenshot below.

![image](https://user-images.githubusercontent.com/33027830/158740772-11e51488-881a-4259-b780-ffc020d3becf.png)

**Comment by rohit-squirrel** [comment]
Yes please provide some doc, will be very helpful @amanmahajan7

**Comment by godon019** [comment]
Hello, you can check the example code here https://github.com/adazzle/react-data-grid/blob/main/website/demos/HeaderFilters.tsx
